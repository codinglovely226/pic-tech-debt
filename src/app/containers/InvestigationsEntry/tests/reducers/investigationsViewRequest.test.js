/* eslint-disable */

import investigationsViewRequestReducer from '../../reducers/investigationsViewRequest';

import {
  resetInvestigationsViewRequest,
  updateInvestigationsViewRequest,
} from '../../actions/case';

describe('investigationsViewRequestReducer', () => {
  it('returns the initial state', () => {
    expect(investigationsViewRequestReducer(undefined, {})).toMatchSnapshot();
  });
  it('reset investigations view', () => {
    expect(investigationsViewRequestReducer({'content': 'Test content'}, resetInvestigationsViewRequest())).toMatchSnapshot();
  });
  it('update investigations view request', () => {
    expect(investigationsViewRequestReducer({}, updateInvestigationsViewRequest('Test Content'))).toMatchSnapshot();
  });
});
