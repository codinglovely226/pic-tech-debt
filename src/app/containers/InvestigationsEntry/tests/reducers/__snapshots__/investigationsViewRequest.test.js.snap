exports[`investigationsViewRequestReducer reset investigations view 1`] = `
Object {
  "content": "",
}
`;

exports[`investigationsViewRequestReducer returns the initial state 1`] = `
Object {
  "content": "",
}
`;

exports[`investigationsViewRequestReducer update investigations view request 1`] = `
Object {
  "content": "Test Content",
}
`;
